hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -176,11 +179,10 @@
   ]
 
   // List of regressions.
-  // TODO #2405: Waiting for bug fixes in Ibex.
-  // regressions: [
-  //   {
-  //     name: sanity
-  //     tests: ["chip_uart_tx_rx"]
-  //   }
-  // ]
+  regressions: [
+    {
+      name: sanity
+      tests: ["chip_uart_tx_rx"]
+    }
+  ]
```

Comment:
Is this intentional? It seems that issue #2405 is still open.

Reply:
Yes, I wanted to check if these changes are working in the private CI. I don't intend to merge this (will be a separate PR).